### DISCUSSION TITLE
Gordian Developer Meeting in July

### DISCUSSION THREAD

**Discussion by shannona** [BlockchainCommons/Gordian-Developer-Community#113] [General & Announcements]
Hey Folks,

This is a reminder that our upcoming Gordian Developer Meeting for July will be held at a European-friendly time: on Wednesday July 5th at 10am PDT, which is 1pm EDT and 7pm CET.

We'd love to hear you talk about your work! Let us know if you'd like to make a brief presentation of 5 or so minutes about your current work, current questions, or other topics of interest.

Some things that we're considering talking about currently are:

* Gordian SeedTool 1.6 — Our demo for CSR
* URs & PSBTs — Recent docs
* Self-Sovereign Identity — A foundational technology

We hope to see you there!

The Blockchain Commons Team

**Comment by thunderbiscuit** [comment]
If you'd have me I can talk a little bit about my experience looking into the UR spec, the libraries for Kotlin Multiplatform I am building, the proposal for animated QR codes in Cashu, and the Bitcoin Development Kit Rust language bindings (and our approach to the scope for these bindings and other libraries in the Rust ecosystem).

**Comment by ChristopherA** [comment]
Yes, we’d love some discussion on these. 

The current draft agenda & slides are at https://hackmd.io/RvMrtTawQ0uieYFeUe3xVg?both, where you you like to fit?

**Reply by thunderbiscuit** [reply]
Yes slide 6 sounds perfect! Thank you.

**Comment by shannona** [comment]
Hey @thunderbiscuit I've got you laid in to talk after Christopher discusses the foundation of UR and some of our new docs:
https://hackmd.io/RvMrtTawQ0uieYFeUe3xVg#/7

I assume you're on either the Mailing List or Signal, and so are seeing the invites to the meeting? If not, let me know ASAP and I'll get you the link.

See you tomorrow (Wednesday) at 10am PDT.

**Reply by thunderbiscuit** [reply]
Yep this is perfect thank you!

On another note I am on the mailing list, however, unlike the June meeting, the July meeting email (the one sent on June 27) doesn't include a link/invite. Am I missing an email maybe?

**Comment by shannona** [comment]
Great. We'll see you tomorrow.

(And the link should be the same as in June, but we'll send it out again in the next few hours.)

**Comment by shannona** [comment]
Just a reminder to everyone that the meeting is tomorrow (Wednesday July 5th) at 10am PDT/1pm EDT/7pm CET.

- Return to URs & Animated QRs
- UR Experiences (thunderbiscuit)
- A LifeHash Use Case (Craig)
- IETF 117 Meetings
- Self-Sovereign Identity
- Gordian SeedTool 1.6 (Wolf)
- Standardizing for CSR

The URL was sent to our Gordian Developer Signal chat and announcements list!
